DNN Platform/Tests/DotNetNuke.Tests.Web/Api/BasicAuthMessageHandlerTests.cs

```diff
@@ -75,6 +75,47 @@ public void ResponseWithNullRequestReturnsUnauthorized()
             // Assert
             Assert.AreEqual("Basic", response.Headers.WwwAuthenticate.First().Scheme);
             Assert.AreEqual("realm=\"DNNAPI\"", response.Headers.WwwAuthenticate.First().Parameter);
+        }
+
+        [Test]
+        public void DoesNotChangeAllowedOriginIfNotConfigured()
+        {
+            // Arrange
+            var response = new HttpResponseMessage(HttpStatusCode.OK) { RequestMessage = null };
+            response.Headers.Add("Access-Control-Allow-Origin", "*");
+            var handler = new BasicAuthMessageHandler(true, false, "", "", null);

```

Comment:
The default for the origins argument would be an empty array instead of `null`, right?
```suggestion
            var handler = new BasicAuthMessageHandler(true, false, "", "", Array.Empty<string>());
```